### PR TITLE
CNV-7293: RN for stable channel

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-7293
Preview build: http://file.rdu.redhat.com/pousley/110220/cnv-7293-rn/virt/virt-2-5-release-notes.html#virt-2-5-new

### CODE REVIEW

File: virt/virt-2-5-release-notes.adoc [hunk 1]
```diff
@@ -202,8 +206,8 @@ drain if there are virtual machines running on top of them.
 * If the {VirtProductName} storage PV is not suitable for importing a RHV VM, the progress bar remains at 10% and the import does not complete. The VM Import Controller Pod log displays the following error message: `Failed to bind volumes: provisioning failed for PVC`. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
 
 //This defect is ON_QA
-* If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. link:https://bugzilla.redhat.com/show_bug.cgi?id=1854425[*BZ#1854425*]
-
+* If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. link:https://bugzilla.redhat.com/show_bug.cgi?id=1854425[(*BZ#1854425*)]
```
Comment (sjhala-ccs) [inline virt/virt-2-5-release-notes.adoc:209]: There is a difference in the color of the parenthesis in the rendered output based on whether they are around the reference text vs enclosing the entire link (as in some of the other known issues). Can you change the other links as well, just for consistency? Thanks!

File: virt/virt-2-5-release-notes.adoc [hunk 2]
```diff
@@ -106,6 +109,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //Placeholder for new content.
 
 //CNV-7293 - Use OLM stable channel for installation
+* There is a new *Update Channel* called *stable*. You must select the *stable* option during installation or upgrade so that {product-title} automatically consumes the correct version of {VirtProductName}.
```
Comment (sjhala-ccs) [inline virt/virt-2-5-release-notes.adoc:112]: Didn't realize that I included a placeholder for this story in two places. Maybe just have it here in the notable technical changes section?

**Review by sjhala-ccs — COMMENTED** [review]
Just a couple of things, but otherwise LGTM!

File: virt/virt-2-5-release-notes.adoc [hunk 3]
```diff
@@ -113,6 +113,7 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 //Placeholder for new content.
 
 //CNV-7293 - Use OLM stable channel for installation
+* When you xref:../virt/install/installing-virt-web.adoc#installing-virt-web[install] or xref:../virt/upgrading-virt.adoc#upgrading-virt[upgrade] {VirtProductName}, you select an *Update Channel*. There is a new *Update Channel* option that is named *stable*. Select the *stable* channel to ensure that you install or upgrade to the version of {VirtProductName} that is compatible with your {product-title} version.
```
Comment (tiraboschi) [inline virt/virt-2-5-release-notes.adoc:116]: This is fine, not sure if the rest was intended as part of this PR

**Pull request merged**